sessions/js-basics/demo-start/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>JS Basics: Demo</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <script src="./js/index.js" defer></script>
+  </head>
+  <body>
+    <h1>JS Basics: Demo</h1>
+    <main class="main" id="main" data-js="main">
```

Comment:
Do we really need the `id` here?

```suggestion
    <main class="main" data-js="main">
```